### PR TITLE
Add service query support for windows guest

### PR DESCRIPTION
**Pull request by xiaodwan**
Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2019.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1121.86 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh Please help to review this PR, thanks.

File: virttest/utils_v2v.py [hunk 1]
```diff
@@ -978,6 +978,15 @@ def get_viostor_info(self):
         cmd = r"dir %s\Drivers\VirtIO\\viostor.sys" % self.windows_root
         return self.run_cmd(cmd)[1]
 
+    def get_service_info(self, name):
```
Comment (chunfuwen) [inline virttest/utils_v2v.py:981]: miss parameter 'name' doc string

Comment (xiaodwan) [reply virttest/utils_v2v.py]: Done, thanks

**Pull request merged**